joomla/joomla-cms
# [4.0] CSS-Grid improvements

Pull Request for Issue #19224 

### Summary of Changes

This PR tweaks the css-grid by altering the scope and ensuring we don't target child elements on the `<body>`, but instead target a class.

This means that we don't affect any potential elements that get appended to the DOM such as alerts, modals, tooltips, etc.

### Testing Instructions

- In the front-end, go to Template Settings
- Click the "Select" button on the Media field

A modal will appear and the dark backdrop should be 100% width of the viewport
